benches/coprocessor_executors/util/fixture.rs

```diff
@@ -279,11 +279,11 @@ impl FixtureBuilder {
                 let mut ci = ColumnInfo::default();
                 ci.set_column_id(index as i64);
                 ci.as_mut_accessor()
-                    .set_tp(ft.tp())
-                    .set_flag(ft.flag())
-                    .set_flen(ft.flen())
-                    .set_decimal(ft.decimal())
-                    .set_collation(ft.collation());
+                    .set_tp(FieldTypeAccessor::tp(&ft))
```

Comment:
Why do we need this change?

Reply:
Because Prost generates methods with the same name on these message types, so we have to be explicit that we want to call the methods from the `FieldTypeAccessor` trait

Reply:
For this purpose, prefer to be:

```rust
let ft = ft.as_accessor();
...
ci.as_mut_accessor()
    .set_tp(ft.tp())
    .set_flag(ft.flag())
    .set_flen(ft.flen())
...
```